sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Release History
-## 7.3.0-beta.3 (Unreleased)
-### New Features
-- Added support receiving dead-letter queue (DLQ) for 'ServiceBusProcessorClient' and session receiver using 'subQueue'
+
+## 7.3.0 (2021-07-08)
+### Features Added
+- Added support for receiving dead-letter queue (DLQ) for `ServiceBusProcessorClient` and session receiver using `subQueue`
+- Added support for configuring `maxAutoLockRenewDuration` for `ServiceBusProcessorClient` in `ServiceBusClientBuilder`.
+- Added support for using `AzureSasCredential` and `AzureNamedKeyCredential` to access a service bus in `ServiceBusClientBuilder`.
+- Exposing `ServiceBusClientBuilder.crossEntityTransaction()` for cross entity transactions support.
+- Exposing `AmqpMessageBody#value` and `AmqpMessageBody#sequence` to support value and sequence amqp types.
+
+### Bugs Fixed
+- Fixed a bug that does not create respect properties when creating Subscription Rule with CorrelationFilter. #21299
```

Comment:
I'd add links to the issues rather than just the number.

Reply:
Added the link.